en-US/GetStarted/Prototype/SetupDevice.htm

```diff
@@ -27,7 +27,7 @@ <h3 class="c-heading-3">Set up your device</h3>
                <br><br><h2>SD card-based devices</h2>
                <p class="c-paragraph-3">SD card-based devices include <b>Raspberry Pi 2 and Raspberry 3 boards</b>, <b>MinnowBoard Turbot/MAX boards</b>, and <b>Intel Compute Sticks</b>. For Intel Compute Sticks, use the Minnowboard Turbot/MAX FFU. You may also need to download a few drivers (such as Wi-Fi and Bluetooth) externally.
                 <br><br>
-            For any of these devices, please ensure that you have the <b>latest firmware update</b>. The latest 64-bit firmware for Minnowboard Turbot/MAX boards can be found on <a class="c-hyperlink" href="https://minnowboard.org/tutorials/updating-the-firmware" target="_blank">the MinnowBoard website</a> (skip step 4). </p><br><br>
+            For any of these devices, please ensure that you have the <b>latest firmware update</b>. The latest 64-bit firmware for Minnowboard Turbot/MAX boards can be found on <a class="c-hyperlink" href="https://minnowboard.org/tutorials/installing-windows-iot-core" target="_blank">the MinnowBoard website</a> (skip step 4). </p><br><br>
```

Comment:
These look like instructions to install IoT Core and not update the firmware. Maybe this is where we should direct them :https://minnowboard.org/tutorials/updating-the-firmware